### PR TITLE
My CV not actualiced

### PR DESCRIPTION
**Pull request by daniscript**

### CODE REVIEW

File: staff/daniel-ferrandez-latre/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                            </address> 
+                        </li>
+                        <li>638078908</li>
+                        <li>daniel.sflatre@gmail.com</li>
+                    </ul>
+                </section>
+                <section>
+                    <picture>
+                        <img src="https://avatars2.githubusercontent.com/u/47394894?s=460&v=4" type="">
+                        </picture>
```
Comment (lexbarba) [inline staff/daniel-ferrandez-latre/cv/index.html]: @DaniLatreCodes  `</picture>`closing tag is not well indented. Please fix. Thanks.

File: staff/daniel-ferrandez-latre/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                    </article>
+                    <article>
+                            <H4>Titulo de grado medio de preimpresión</H4>
+                            <section>
+                                <p>Cursado en “Centre Educa# u Roca“ Barcelona.</p>
+                                <p>Conocimientos</p>
+                                <ol>
+                                    <li>Maquetación, fotomontage, diferentes formatos de impresión.</li>
+                                    <li>Programas de la gama adobe (versión CC 2019)</li>
+                                    <ul>
```
Comment (lexbarba) [inline staff/daniel-ferrandez-latre/cv/index.html]: @DaniLatreCodes and ol or ul tag can only have li as child elements. for nested lists, they should be inside a li tag. please fix.

File: staff/daniel-ferrandez-latre/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                                    <li>Mantenimiento de la web recibiendo comandas del cliente. </li>
+                                    <li>Modificación de diferentes webs WordPress tanto por entorno gráﬁ  co como por html y css.</li>
+                                    <li>Web Sagrada Familia,  Federació Catalana contra el cancer. </li>
+                                    <li>Ayuda en el mantenimiento de la Web TYSCA en .NET.</li>
+                                </ol>
+                            </section>
+                    </article>
+             
+                    <article>
+                            <H3>2014</H3>
```
Comment (lexbarba) [inline staff/daniel-ferrandez-latre/cv/index.html]: @DaniLatreCodes no capital letters inside tag. please fix them all. thanks.

**Review by lexbarba — CHANGES_REQUESTED** [review]

File: staff/daniel-ferrandez-latre/cv/index.html (hunk 1 above)
Comment (daniscript) [reply staff/daniel-ferrandez-latre/cv/index.html]: Done!

File: staff/daniel-ferrandez-latre/cv/index.html (hunk 2 above)
Comment (daniscript) [reply staff/daniel-ferrandez-latre/cv/index.html]: Done!

File: staff/daniel-ferrandez-latre/cv/index.html (hunk 3 above)
Comment (daniscript) [reply staff/daniel-ferrandez-latre/cv/index.html]: Done!

**Pull request merged**